gitlab/mixins.py

```diff
@@ -298,6 +298,9 @@ def save(self, **kwargs):
             GitlabUpdateError: If the server cannot perform the request
         """
         updated_data = self._get_updated_data()
+        # Nothing to update. Servers fails if sent an empty dict.
+        if len(updated_data.keys()) == 9:
```

Comment:
Could you explain the logic here? Where does the 9 come from?

Thanks.

Reply:
Whops, typo when creating the patch. It should be a 0.